### PR TITLE
contributors/devel: document '// +groupGoName' code-generation tag

### PR DESCRIPTION
**Pull request by sttts**
Documentation counterpart PR for https://github.com/kubernetes/kubernetes/pull/54950.

### CODE REVIEW

File: contributors/devel/api_changes.md [hunk 1]
```diff
@@ -544,6 +544,10 @@ Once you added the annotations, generate the client with
 hack/update-codegen.sh
 ```
 
+Note that you can use the optional `// +groupGoName=` to specify a CamelCase
+custom Golang identifier to de-conflict e.g. `policy.authorization.k8s.io` and
+`policy.k8.sio`. These two would both map to `Policy()` in clientsets.
```
Comment (mikedanese) [inline contributors/devel/api_changes.md]: typo: `k8.sio`

Comment (sttts) [reply contributors/devel/api_changes.md]: fixed

**Comment by sttts** [comment]
kubernetes/kubernetes#54950 merged. Merging here as well.

**Pull request merged**